modules/n1ql_fts/pages/index.adoc

```diff
@@ -0,0 +1,589 @@
+= How to use Full-Text Search with N1QL
+
+
+We already discussed how the link:https://blog.couchbase.com/why-you-should-avoid-like-deep-dive-on-fts-part-1/[Like %] clause is potentially harmful for your application at scale, and why you should consider using Couchbase Full-Text Search instead. But one thing we haven't mentioned yet is why you might prefer Couchbase FTS over any other FTS Engine in the market.
+
+Apart from the fact that using Couchbase FTS is simpler in terms of setup and infrastructure, as you can leverage the infrastructure you already have for your database. You can combine the power of N1QL and FTS in a single statement:
+
+
+[source,SQL,indent=0]
```

Comment:
You can use `[source,N1QL]` for source code formatting in our docs — the language is identified when you hover over the code.